portfolio/src/main/webapp/script.js

```diff
@@ -35,3 +35,42 @@ async function showString() {
     const stringContainer = document.getElementById('string-container');
     stringContainer.innerText = textFromResponse;
 }
+
+function createMap() {
+    const map = new google.maps.Map(
+        document.getElementById('map'),
+        {center: {lat: 40.76060063892388, lng:-73.98365302868159}, zoom: 12});
+    
+    const Birch = new google.maps.Marker({
```

Comment:
nit: be consistent with the naming patterns of your consts. all should start lowercase. ie here this should be "birch" rather than "Birch"